test/e2e/accounts/create-snap-account.spec.ts

```diff
@@ -101,9 +103,9 @@ describe('Create Snap Account', function () {
         fixtures: new FixtureBuilder().build(),
         ganacheOptions: defaultGanacheOptions,
         failOnConsoleError: false,
-        title: this.test.fullTitle(),
+        title: this.test?.fillTitle,
```

Comment:
what is 'fillTitle' ? and is this.test ever not defined?

Reply:
Haha whoops, merge error!  If you don't put in that `?.` the linter complains.  But I don't know why it's ever undefined.